app/daemon.go

```diff
@@ -109,6 +114,8 @@ func startManager(c *cli.Context) error {
 
 	defaultSettingPath := os.Getenv(types.EnvDefaultSettingPath)
 
+	types.SetProfilerDisabled(c.Bool(FlagDisableProfiler))
```

Comment:
Let's get the falg and pass it to the server directly instead of saving in another place. 

```
server := api.NewServer(m, wsc, profileEnabled).
```

Reply:
Addressed.